### ISSUE TITLE
Feature request: unable double tap

### ISSUE THREAD

**Issue by jofuelo** [MikeOrtiz/TouchImageView#531]
It would be nice and easy to implement to add a method enableDoubleTap(boolean enable) to enable or disable the default double tap behaviour. It would be easy with just a boolean flag.

**Comment by hannesa2** [comment]
Yes, this would be nice https://github.com/MikeOrtiz/TouchImageView?tab=readme-ov-file#project-status-maintenance-mode
